repo/packages/M/minio/10/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "packagingVersion": "3.0",
+  "name": "minio",
+  "version": "0.0.11-RELEASE.2017-11-22T19-55-46Z",
+  "minDcosReleaseVersion": "1.8",
+  "scm": "https://github.com/minio/minio.git",
+  "maintainer": "dev@minio.io",
+  "website": "https://www.minio.io",
+  "framework": false,
+  "description": "Minio is an object storage server for cloud native applications. To deploy at scale, use DC/OS to orchestrate Minio containers one per tenant.",
+  "tags": [
+    "data",
+    "storage",
+    "filesystem"
+  ],
+  "preInstallNotes": "This DC/OS service is currently in preview. There may be bugs, incomplete features, incorrect documentation or other discrepancies in packaging. Preview packages should never be used in production! There is no limit to number of Minio services you can launch. To learn how to install Minio on DC/OS refer to https://github.com/dcos/examples/blob/master/minio/1.9/README.md",
```

Comment:
@krisis as discussed with @ryadav88 , we can update the `preInstallNotes` to say 

```
This is a community package. Community packages are unverified and unreviewed content from the community.
```

Reply:
@nitisht, @ryadav88  I have updated the preInstallNotes. PTAL

Reply:
Thanks @krisis LGTM